code/datums/components/unbreakable.dm

```diff
@@ -0,0 +1,31 @@
+/datum/component/unbreakable
+	COOLDOWN_DECLARE(surge_cooldown)
+
+/datum/component/unbreakable/Initialize()
+	if(!ishuman(parent))
+		return COMPONENT_INCOMPATIBLE
+	ADD_TRAIT(parent, TRAIT_UNBREAKABLE, INNATE_TRAIT)
+
+/datum/component/unbreakable/Destroy(force, silent)
+	. = ..()
+	REMOVE_TRAIT(parent, TRAIT_UNBREAKABLE, INNATE_TRAIT)
+
+/datum/component/unbreakable/RegisterWithParent()
+	RegisterSignal(parent, COMSIG_MOB_STATCHANGE, .proc/surge)
+
+/datum/component/unbreakable/UnregisterFromParent()
+	UnregisterSignal(parent, COMSIG_MOB_STATCHANGE)
+
+/datum/component/unbreakable/proc/surge(mob/living/carbon/human/surged, new_stat)
+	SIGNAL_HANDLER
+	if(new_stat != UNCONSCIOUS)
```

Comment:
```suggestion
	if(new_stat < SOFT_CRIT)
```

Reply:
is the intention here for this to trigger as soon as you enter crit? or is it for it to only trigger if you're in deep crit? either way, if you're just checking to see if their stat is exactly UNCONSCIOUS, I don't think that this will trigger if you just fall into SOFT_CRIT or HARD_CRIT

Reply:
makes sense to me, but i don't want it triggering on soft crit as it'll get in the way of tenacity

Reply:
> makes sense to me, but i don't want it triggering on soft crit as it'll get in the way of tenacity

Okay, I've adjusted my suggestion to account for this.